nowriter/tmencoding/tm_encoder_columnar_builder.go

```diff
@@ -0,0 +1,23 @@
+package tmencoding
+
+import "time"
+
+// Simplest stateful builder interface with chaining disallowed.
```

Comment:
do we want both chaining and no chaining? maybe we should be consistent in style?